pkg/acquisition/acquisition.go

```diff
@@ -242,14 +247,17 @@ func StartAcquisition(sources []DataSource, output chan types.Event, AcquisTomb
 				err = subsrc.OneShotAcquisition(output, AcquisTomb)
 			}
 			if err != nil {
-				return err
+				//if one of the acqusition returns an error, we kill the others to properly shutdown
+				AcquisTomb.Kill(err)
 			}
 			return nil
 		})
 		//register acquisition specific metrics
 		prometheus.MustRegister(subsrc.GetMetrics()...)
 	}
+	log.Infof("waiting on tomb...")
```

Comment:
I would use log sth like "waiting on acquisition tomb" or maybe "waiting for acquisition to terminate"